### PR TITLE
Implemented the api for fetching list of tasks that are already executed

### PR DESCRIPTION
**Pull request by danishjamal104**
Implemented the API for fetching the list of tasks that are already completed, by the `processDefinitionKey`.

*Sample request*
```localhost:5000/channel/task/2251799813686414```

*Sample response*
```JSON
{
  "tasks": []
}
```

### CODE REVIEW

File: src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java [hunk 1]
```diff
@@ -39,6 +42,58 @@
...
+         * sample url:
+         * localhost:5000/channel/task/2251799813686414
+         *
+         * sample response: {
+         *   "tasks": [
+         *     ""
+         *   ]
+         * }
+         */
+        from(String.format("rest:get:/channel/task/{%s}", PROCESS_DEFINITION_KEY))
```
Comment (SubhamPramanik) [inline src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java:57]: Please follow a consistent API spec:
/category/{category_id}/sub-category

So this should be:
/process/{processId}/task

File: src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java [hunk 2]
```diff
@@ -285,4 +340,8 @@ public void configure() {
                 .setBody(constant(null));
 
     }
+
+    private QueryBuilder getMatchPhraseQueryBuilder(String key, Object valueToMatch) {
```
Comment (SubhamPramanik) [inline src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java:344]: Is this required to be a separate function for a single line?

**Review by SubhamPramanik — CHANGES_REQUESTED** [review]

File: src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java (hunk 2 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java:344]: it was used at multiple places so I separated it in function, incase any common thing is to be changed at all the places.

Comment (SubhamPramanik) [reply src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java:344]: If it's only going to be a single line of code in the function, then please do.

File: src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java (hunk 1 above)
Comment (SubhamPramanik) [reply src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java]: LGTM

**Comment by SubhamPramanik** [comment]
Looks fine. Small change and resolve conflicts. Thanks.

**Pull request merged**